### PR TITLE
Slack - Add possibility to use the bot's token in the OAuth2 credentials

### PR DESCRIPTION
**Pull request by RicardoE105**

### CODE REVIEW

File: packages/nodes-base/credentials/SlackOAuth2Api.credentials.ts [hunk 1]
```diff
@@ -58,5 +62,67 @@ export class SlackOAuth2Api implements ICredentialType {
 			type: 'hidden',
 			default: 'body',
 		},
+		{
+			displayName: `Use Bot's Access Token`,
+			name: 'useBotToken',
+			type: 'boolean',
+			default: false,
+			description: `Weather to use the bot's access token or user's access token`,
```
Comment (pemontto) [inline packages/nodes-base/credentials/SlackOAuth2Api.credentials.ts:70]: `Weather` => `Whether` 👍

**Review by michael-radency — DISMISSED** [review]

**Comment by mendoncart** [comment]
Is this solved?

**Comment by xenothomas** [comment]
> Is this solved?

I also ended up here as a result of my search to what to do to get the option to post as a bot although "Bot Token Scopes" have been added in the Slack OAuth & Permissions

**Comment by RicardoE105** [comment]
@Joffcom

**Comment by reikjarloekl** [comment]
@RicardoE105 Just curious as to why this was not merged but is closed. Is there another way to post to slack using as bot instead of user when using OAuth2 credentials?

**Comment by RicardoE105** [comment]
If I remember correctly, when you create an app on Slack, you get two tickets, a user one and a bot one. The bot one can be used on the node’s acesss token authetication. @Joffcom is this still the case?

**Pull request closed without merging**